releasenotes/notes/trace-agent-appsec-events-obfuscation-a4c1705aa766475c.yaml

```
@@ -0,0 +1,17 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    AppSec: obfuscate security events. It is enabled by default so that it obfuscates
```

```suggestion
    APM: obfuscate AppSec security events. It is enabled by default so that it obfuscates
```
The reason we're putting `APM` is to be able to differentiate between `trace-agent` changes and other changes (`trace-agent` used to be a separate repository). I don't know if this is still a valid point. Perhaps there should be another field somewhere. But for now, we have to use `APM`.